packages/chain-adapters/src/utxo/bitcoin/BitcoinChainAdapter.test.ts

```diff
@@ -547,4 +547,56 @@ describe('BitcoinChainAdapter', () => {
       expect(res).toMatchObject(expectedReturnValue)
     })
   })
+
+  describe('getBIP44Params', () => {
+    const adapter = new bitcoin.ChainAdapter(args)
+    it('should throw for undefined accountType', async () => {
+      expect(() => {
+        adapter.getBIP44Params({ accountNumber: 0, accountType: undefined })
+      }).toThrow('not a supported accountType undefined')
+    })
+    it('should always be coinType 0', async () => {
+      for (const acctType of adapter.getSupportedAccountTypes()) {
+        const r = adapter.getBIP44Params({ accountNumber: 0, accountType: acctType })
+        expect(r.coinType).toStrictEqual(0)
+      }
+    })
+    it('should properly map account types to purposes', async () => {
+      const accountTypes: UtxoAccountType[] = [
```

Comment:
best for tests to get data from the single source of truth here https://github.com/shapeshift/lib/blob/add-getBIP44Params/packages/types/src/base.ts#L25

Reply:
hmmm it is using them, I just enumerated the values so it isn't dependent on ordering which it doesn't need to be